common/app/assets/assets.scala

```diff
@@ -142,17 +142,21 @@ class Assets(base: String, assetMap: String = "assets/assets.map") extends Loggi
   }
 
   object js {
-     private def inlineJs(path: String): String = IOUtils.toString(AssetFinder(path))
+     private def contents(path: String): String = IOUtils.toString(AssetFinder(path))
```

Comment:
The usage of this function on [line 154](https://github.com/guardian/frontend/pull/10327/files#diff-aef18c02671cecab73288a55f9311a39R154) isn't for inlining JS, it's simply used for reading the file contents (and then parsing that as JSON).